pkg/pipelines/config/config.go

```diff
@@ -22,10 +21,21 @@ func PathForEnvironment(env *Environment) string {
 	return filepath.Join("environments", env.Name)
 }
 
+//PathForCICDEnvironment returns the path only for the CICD environment
+func PathForCICDEnvironment(cicd *Cicd) string {
```

Comment:
If CICD and ArgoCD are no longer Environments, we should probably stop calling them "Environments".

Reply:
Changed to PathForArgoCdConfig() and PathForCICDConfig().